packages/components/src/notice/style.scss

```diff
@@ -3,7 +3,7 @@
 	font-family: $default-font;
 	font-size: $default-font-size;
 	background-color: $white;
-	border-left: 4px solid var(--wp-admin-theme-color);
+	border-left: 4px solid $components-color-accent;
```

Comment:
This is an interesting case for theming... on some themes (like the "Sunset" theme we're currently using in Storybook) the notices can take on an orange color. With another theme it could even be red. These colors on notices usually have more meaning than they would in other components, indicating the severity/urgency of the notice.

Not sure if/how that impacts how we'd want to apply theming on this component.

Reply:
That's an interesting point. I will note that the `success`, `warning` and `error` statues of notice already have a hardcoded border color (a few lines below in this file), and so this specific rule would only affect the "default" and 
"informational" statuses.

Current behaviour is also explained in [the README](https://github.com/WordPress/gutenberg/blob/9efab2cbe695ff831c0c8b099b3193629c4989a7/packages/components/src/notice/README.md#usage) — we should keep that in mind in case it needs updating too

Reply:
> Current behaviour is also explained in [the README](https://github.com/WordPress/gutenberg/blob/9efab2cbe695ff831c0c8b099b3193629c4989a7/packages/components/src/notice/README.md#usage) — we should keep that in mind in case it needs updating too

Good call. I've added an update to the README ✅